source/uwp/Renderer/lib/Util.cpp

```diff
@@ -92,6 +92,12 @@ std::string HStringToUTF8(winrt::hstring const& in)
     return WStringToString(in);
 }
 
+double HStringToDouble(winrt::hstring const& in)
+{
+    std::string inAsUTF8 = HStringToUTF8(in);
+    return std::atof(inAsUTF8.c_str());
```

Comment:
>atof [](http://example.com/codeflow?start=16&length=4)

Can use std::stof to avoid the need to convert wide to utf8?

[https://en.cppreference.com/w/cpp/string/basic_string/stof](https://en.cppreference.com/w/cpp/string/basic_string/stof)

Reply:
good call out - thanks!